styles/dialog/fight.scss

```diff
@@ -23,9 +23,64 @@
         display: grid;
         width: 100%;
         height: min-content;
+        overflow: hidden;
```

Comment:
Ditto with this change. This hides any part of the grid that displays the fight cards if it happens to overflow its' maximum height. It doesn't have a maximum height though.